shell.nix

```diff
@@ -3,4 +3,4 @@
 
 (import (fetchTarball https://github.com/edolstra/flake-compat/archive/master.tar.gz) {
   src = builtins.fetchGit ./.;
-}).shellNix
+}).defaultNix.devShell
```

Comment:
This seems strange. How can `defaultNix` have a `devShell` attribute? Also `shellNix` is defined as `devShell.${system}` in flake-compat.

Reply:
i frankly don't know how the flake setup really works. The way this came to be is that we added an additional shell instead of using the default one.

Looking at the current status I think I could just add `foreman` as buildInput to the default shell and then i can just drop this and use `shellNix` again.